tools/elf2bin.py

```diff
@@ -30,6 +30,9 @@
 ffreqb = { '40': 0, '26': 1, '20': 2, '80': 15 }
 fsizeb = { '512K': 0, '256K': 1, '1M': 2, '2M': 3, '4M': 4, '8M': 8, '16M': 9 }
 
+crcsize_offset = 4088
+crcval_offset = 4092
```

Comment:
Hm. I think it is off by one error, starting index should be 4087?

Reply:
@mcspr Build+upload with crcsize_offset = 4087 crashes on boot.

Reply:
Right, nvm :)